libs/frontend/feature-build/src/lib/dnd/row-layout.ts

```diff
@@ -18,7 +18,7 @@ interface IHandleHoverProps {
   targetMiddle: XYCoord
   haveSameParent: boolean
   dragComponentIndex: number
-  targetIndex: number
+  hoverCmpIndex: number
```

Comment:
The changes made to this file are just refactoring (prop renaming, destructing assignment, etc), the logic stays exactly the same.